src/govuk/i18n.mjs

```diff
@@ -371,10 +374,3 @@ I18n.pluralRules = {
  * @property {string} [few] - Plural form used for a few
  * @property {string} [many] - Plural form used for many
  */
-
-/**
- * Translated messages (flattened)
- *
- * @private
- * @typedef {Object<string, string> | {}} TranslationsFlattened
```

Comment:
Removed as config objects from `extractConfigByNamespace()` have mixed values

(Not just strings)